packages/integration-tests/src/resolvers/mutations/bookAdvance/saveBookAdvanceResolver.test.ts

```diff
@@ -0,0 +1,20 @@
+import { Context } from "src/context";
+import { SaveBookAdvanceInput } from "src/generated/graphql-types";
+import { run } from "src/resolvers/testUtils";
+import { saveBookAdvance } from "src/resolvers/mutations/bookAdvance/saveBookAdvanceResolver";
+
+import "src/setupDbTests";
+
+describe("saveBookAdvance", () => {
+  it.withCtx("can create", async (ctx) => {
+    const { em } = ctx;
+    const result = await runSaveBookAdvance(ctx, () => ({}));
+    // const ba = await em.load(BookAdvance, result.BookAdvance);
```

Comment:
Commenting out b/c these aren't (currently) supposed to do test anything other than the code generation works as expected.